### PR TITLE
Add parameters (patience, max_sequence_length, and batch_size) for sequence labelling command line scripts 

### PR DESCRIPTION
**Pull request by lfoppiano**
This PR adds patience, batch_size and max_sequence_length when not already present, in the applications script using sequence labelling.

### CODE REVIEW

File: delft/applications/nerTagger.py [hunk 1]
```diff
@@ -181,7 +190,7 @@ def train_eval(embeddings_name=None,
                 use_ELMo=False): 
 
     batch_size, max_sequence_length, patience, recurrent_dropout, early_stop, max_epoch, embeddings_name, word_lstm_units, multiprocessing = \
-        configure(architecture, dataset_type, lang, embeddings_name, use_ELMo)
+        configure(architecture, dataset_type, lang, embeddings_name, use_ELMo, max_sequence_length=-1, batch_size=-1, patience=-1)
```
Comment (kermitt2) [inline delft/applications/nerTagger.py:193]: it should be:
```python
configure(architecture, dataset_type, lang, embeddings_name, use_ELMo, max_sequence_length, batch_size, patience)
```
(it need to use the command line parameters)

**Review by kermitt2 — CHANGES_REQUESTED** [review]

**Comment by lfoppiano** [comment]
Thanks @kermitt2, I've amended the changes following your comments

File: delft/applications/grobidTagger.py [hunk 2]
```diff
@@ -128,15 +126,18 @@ def configure(model, architecture, output_path=None, max_sequence_length=-1, bat
...
+        patience = 5
+
+    return batch_size, max_sequence_length, model_name, embeddings_name, max_epoch, multiprocessing, early_stop, patience
 
 
 # train a GROBID model with all available data
 def train(model, embeddings_name=None, architecture=None, transformer=None, input_path=None, 
         output_path=None, features_indices=None, max_sequence_length=-1, batch_size=-1, max_epoch=-1, 
-        use_ELMo=False, incremental=False, input_model_path=None):
+        use_ELMo=False, incremental=False, input_model_path=None, patience=5):
```
Comment (kermitt2) [inline delft/applications/grobidTagger.py:140]: I think it should be here:
```
patience=-1
```
otherwise the command line parameter is never considered.

**Review by kermitt2 — CHANGES_REQUESTED** [review]

**Comment by kermitt2** [comment]
Thank you Luca for your patience for getting me reviewing the PD :D

**Comment by lfoppiano** [comment]
Thank you!! 👍

**Pull request merged**